src/services/panel.ts

```diff
@@ -30,6 +29,27 @@ export function setLeverColorOverrides(overrides: any) {
   });
 }
 
+interface TimeSeriesFieldConfig extends FieldConfig {
+  hideFrom: HideSeriesConfig;
+}
+export function setLevelSeriesOverrides(levels: string[], overrides: FieldConfigOverridesBuilder<FieldConfig>) {
```

Comment:
This override is not setting the visibility of levels in the Logs Volume viz and I don't understand why. Need to further investigate.

Reply:
create a panel in a dashboard where it works and compare the JSON data of your viz

Reply:
Good idea. I applied the same structure and values that Grafana was internally setting, but maybe I missed something.

Reply:
```
"overrides": [
          {
            "matcher": {
              "id": "byName",
              "options": "{level=\"info\"}"
            },
            "properties": [
              {
                "id": "custom.hideFrom",
                "value": {
                  "legend": false,
                  "tooltip": false,
                  "viz": true
                }
              }
            ]
          }
        ]
```

https://obslogs.grafana-dev.net/d/cds8vpood924ga/new-dashboard?tab=queries&viewPanel=panel-1&from=2024-07-19T11:48:02.437Z&to=2024-07-19T17:48:02.437Z

Reply:
That's for one. Hiding multiple:

```
"overrides": [
          {
            "__systemRef": "hideSeriesFrom",
            "matcher": {
              "id": "byNames",
              "options": {
                "mode": "exclude",
                "names": [
                  "{level=\"error\"}"
                ],
                "prefix": "All except:",
                "readOnly": true
              }
            },
            "properties": [
              {
                "id": "custom.hideFrom",
                "value": {
                  "legend": false,
                  "tooltip": false,
                  "viz": true
                }
              }
            ]
          }
        ]
```

Reply:
Ohhhh, it was "byNames" not "byName" ‼️  🙇 @gtk-grafana